component/org.wso2.carbon.identity.oauth2.token.handler.clientauth.tlswithidsecret/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/tlswithidsecret/MutualTLSWithIdSecretAuthenticator.java

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
These were already there previously right? So we need to stick to original year

Reply:
Addressed with https://github.com/wso2-extensions/identity-oauth-addons/pull/71/commits/8349413436c16d7cae29e48a83bbcf640ee30fdc.